modules/lang-painless/spi/src/main/java/org/elasticsearch/painless/spi/annotation/AliasAnnotation.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.painless.spi.annotation;
+
+/**
+ * Creates an alias in PainlessLookupBuilder for the given class.  Can be used to expose an inner class without
+ * scripts and whitelists needing to scoping it by the outer class.
+ *
+ * For class
+ * public class Outer {
```

Comment:
Code snippets should be in a `<pre>` block

Reply:
Added.